src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -970,4 +970,4 @@ interface taskInfo {
 }
 
 type TombstoneItem = Raven.Server.Documents.TombstoneCleaner.StateHolder & { Collection: string };
-type TombstonesStateOnWire = Omit<Raven.Server.Documents.TombstoneCleaner.TombstonesState, "Tombstones"> & { Results: TombstoneItem[] };
+type TombstonesStateOnWire = Omit<Raven.Server.Documents.TombstoneCleaner.TombstonesState, "Tombstones"> & { Results: TombstoneItem[]; PerSubscriptionInfoExtended: subscriptionInfoExtended[] };
```

Comment:
There is no `subscriptionInfoExtended` type here. It's .d.ts file so it will not show the error. From what I can see, in class `TombstonesState` the `Persubcriptioninfoexteded` field is an object but here should be an array. It should look like this:

```
type TombstonesStateOnWire = Omit<
    Raven.Server.Documents.TombstoneCleaner.TombstonesState,
    "Tombstones" | "PerSubscriptionInfoExtended"
> & {
    Results: TombstoneItem[];
    PerSubscriptionInfoExtended: Raven.Server.Documents.TombstoneCleaner.TombstonesState.SubscriptionInfoExtended[];
};
```

Reply:
Done